comp/core/flare/types/types.go

```diff
@@ -21,15 +24,40 @@ type FlareBuilder = flarebuilder.FlareBuilder
 // see the aliased type for the full description
 type FlareCallback = flarebuilder.FlareCallback
 
+// FlareTimeout is called by the FlareBuilder to determine the maximum allowed runtime
+// see the aliased type for the full description
+type FlareTimeout = flarebuilder.FlareTimeout
+
+// FlareArgs contains the args passed in by the caller to the flare generation process
+// see the aliased type for the full description
+type FlareArgs = flarebuilder.FlareArgs
+
+// FlareFiller is a struct containing a callback that provides a chunk of information for flare creation
+// see the aliased type for the full description
+type FlareFiller = flarebuilder.FlareFiller
+
```

Comment:
Similar to https://github.com/DataDog/datadog-agent/pull/30851 let's moves those types here instead of creating aliases.

Reply:
Sold